pkg/envoy/admin/client.go

```diff
@@ -73,7 +73,7 @@ func (a *envoyAdminClient) GenerateAPIToken(dataplane *mesh_core.DataplaneResour
 }
 
 func (a *envoyAdminClient) getOrCreateSigningKey(mesh string) (string, error) {
-	key, err := issuer.GetSigningKey(a.rm, issuer.DataplaneTokenPrefix, mesh)
+	key, err := issuer.GetSigningKey(a.rm, issuer.EnvoyAdminClientTokenPrefix, mesh)
```

Comment:
revert this please. There is a separate PR and it's not that simple change

Reply:
on it

Reply:
done